components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/storage/model/PackageEntity.java

```diff
@@ -66,6 +66,9 @@
     @Column(name = "licence_type", length = 256)
     private String licenseType;
 
+    @Column(name = "support_info", length = 256)
```

Comment:
As this field can have an arbitrary length by spec, I think we can remove the length restriction.